app/templates/rentals/index.hbs

```diff
@@ -0,0 +1,7 @@
+<ul class="results">
+  {{#each model as |rentalUnit|}}
+    {{#link-to "rentals.show" rentalUnit.slug}}
```

Comment:
We can totally PR the CSS addon repo with anything we need to make super-rentals right: https://github.com/toddjordan/ember-cli-tutorial-style